frontend/src/embedding-sdk-bundle/components/public/dashboard/SdkDashboard.tsx

```diff
@@ -385,6 +411,7 @@ SdkDashboard.Tabs = Dashboard.Tabs;
 SdkDashboard.ParametersList = Dashboard.ParametersList;
 SdkDashboard.FullscreenButton = Dashboard.FullscreenButton;
 SdkDashboard.ExportAsPdfButton = Dashboard.ExportAsPdfButton;
+SdkDashboard.SubscriptionsButton = Dashboard.SubscriptionsButton;
```

Comment:
Not sure I need this, but since export pdf button is included, I guess this one should be included too.